diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index 1050af5..2dfbf44 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -608,7 +608,9 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
     const handleOnAnimate = useCallback(
       function handleOnAnimate(toPoint: number) {
         const snapPoints = animatedSnapPoints.value;
-        const toIndex = snapPoints.indexOf(toPoint);
+        const toIndex = snapPoints.indexOf(toPoint) === -1
+          ? snapPoints.indexOf(toPoint + animatedKeyboardHeight.value)
+          : snapPoints.indexOf(toPoint);
 
         print({
           component: BottomSheet.name,
@@ -701,7 +703,9 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
          */
         animatedNextPosition.value = position;
         animatedNextPositionIndex.value =
-          animatedSnapPoints.value.indexOf(position);
+          animatedSnapPoints.value.indexOf(position) === -1
+            ? animatedSnapPoints.value.indexOf(position + animatedKeyboardHeight.value)
+            : animatedSnapPoints.value.indexOf(position);
 
         /**
          * fire `onAnimate` callback
@@ -1167,11 +1171,14 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
       () => ({
         opacity:
           Platform.OS === 'android' && animatedIndex.value === -1 ? 0 : 1,
-        transform: [
-          {
-            translateY: animatedPosition.value,
-          },
-        ],
+        // USE ONLY marginTop!!! For android transform move tree under root content
+        // code commented cause https://jira-eu-aholddelhaize.atlassian.net/browse/APCONV-3757
+        // transform: [
+        //   {
+        //     translateY: animatedPosition.value,
+        //   },
+        // ],
+        marginTop: animatedPosition.value,
       }),
       [animatedPosition, animatedIndex]
     );
@@ -1614,7 +1621,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
               animatedPosition={animatedPosition}
               backdropComponent={backdropComponent}
             />
-            <BottomSheetContainer
+            {/* <BottomSheetContainer
               key="BottomSheetContainer"
               shouldCalculateHeight={!$modal}
               containerHeight={_animatedContainerHeight}
@@ -1623,7 +1630,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
               bottomInset={bottomInset}
               detached={detached}
               style={_providedContainerStyle}
-            >
+            > */}
               <Animated.View style={containerStyle}>
                 <BottomSheetBackgroundContainer
                   key="BottomSheetBackgroundContainer"
@@ -1687,7 +1694,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
                   // isInTemporaryPosition,
                 }}
               /> */}
-            </BottomSheetContainer>
+            {/* </BottomSheetContainer> */}
           </BottomSheetGestureHandlersProvider>
         </BottomSheetInternalProvider>
       </BottomSheetProvider>
